### PR TITLE
WIP: BL-5935: Modified header classes, ensured error templates were consis

### PR DESCRIPTION
**Pull request by Jerotire**
…tent

### CODE REVIEW

File: selenium/src/main/java/uk/gov/dvsa/motr/ui/page/SubscriptionConfirmationErrorPage.java [hunk 1]
```diff
@@ -10,7 +10,7 @@
 @GotoUrl("/confirm-subscription/{0}")
 public class SubscriptionConfirmationErrorPage extends Page {
 
-    @FindBy(className = "heading-xlarge")
+    @FindBy(className = "heading-large")
```
Comment (aaronlong) [inline selenium/src/main/java/uk/gov/dvsa/motr/ui/page/SubscriptionConfirmationErrorPage.java]: I think we can remove this headerTitle variable, the override of selfVerify.  Would also need to update getHeaderTitle method to use title instead.

File: selenium/src/main/java/uk/gov/dvsa/motr/ui/page/UnsubscribeErrorPage.java [hunk 2]
```diff
@@ -10,7 +10,7 @@
 @GotoUrl("/unsubscribe/{0}")
 public class UnsubscribeErrorPage extends Page {
 
-    @FindBy(className = "heading-xlarge")
+    @FindBy(className = "heading-large")
```
Comment (aaronlong) [inline selenium/src/main/java/uk/gov/dvsa/motr/ui/page/UnsubscribeErrorPage.java]: I think we can remove this headerTitle variable, the override of selfVerify. Would also need to update getHeaderTitle method to use title instead.

File: webapp/src/main/resources/template/phone-number.hbs [hunk 3]
```diff
@@ -6,6 +6,8 @@
 
             <a href="{{back_url}}" class="link-back">{{back_button_text}}</a>
 
+            <h1 class="heading-large">What is your mobile number?</h1>
+
```
Comment (mathieuhendey) [inline webapp/src/main/resources/template/phone-number.hbs]: Headers should be below the validation errors. This header is in the wrong place.

Comment (mathieuhendey) [reply webapp/src/main/resources/template/phone-number.hbs]: Also there are now two headers on this page.

File: webapp/src/main/resources/template/sms-confirmation-code.hbs [hunk 4]
```diff
@@ -4,7 +4,7 @@
 
         <div class="column-two-thirds">
 
-            <h1 class="heading-xlarge">One more step</h1>
+            <h1 class="heading-large">One more step</h1>
```
Comment (mathieuhendey) [inline webapp/src/main/resources/template/sms-confirmation-code.hbs]: This heading is also in the wrong place.

**Comment by mathieuhendey** [comment]
Finally, squash commits down to one please.

File: webapp/src/main/resources/template/phone-number.hbs (hunk 3 above)
Comment (Jerotire) [reply webapp/src/main/resources/template/phone-number.hbs]: Missed that!

File: webapp/src/main/resources/template/sms-confirmation-code.hbs [hunk 5]
```diff
@@ -4,8 +4,6 @@
 
         <div class="column-two-thirds">
 
-            <h1 class="heading-xlarge">One more step</h1>
-
             <p>We sent you a text message with a 6-digit code. Enter the code below to activate your reminder.</p>
```
Comment (mathieuhendey) [inline webapp/src/main/resources/template/sms-confirmation-code.hbs]: Move this `<p>` back below the header

**Pull request merged**